src/man/sssd-kcm.8.xml

```diff
@@ -189,7 +189,7 @@ tgt_renewal_inherit = domain-name
             options are described in detail below
             <programlisting>
 krb5_renew_interval
-krb5_renew_lifetime
+krb5_renewable_lifetime
```

Comment:
But `krb5.conf` option name is `renew_lifetime`.
@justin-stephenson, could you please take a look?

Reply:
Well, SSSD option is indeed named `krb5_renewable_lifetime`:
https://github.com/SSSD/sssd/blob/fc3fad982e39d560a80c1a8b922455a190718cb7/src/confdb/confdb.h#L287